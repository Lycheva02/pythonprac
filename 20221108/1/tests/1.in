a = DivStr("109js109j1010j")
print(* a//5)
print(a % 7)