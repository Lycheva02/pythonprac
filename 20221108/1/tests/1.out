10 9j s1 09 j1

